tide_alert.install

```diff
@@ -7,18 +7,38 @@
 
 use Drupal\taxonomy\Entity\Term;
 use Drupal\workflows\Entity\Workflow;
+use Drupal\user\Entity\Role;
+
+/**
+ * Install default set of alert types.
+ */
+function _tide_alert_create_default_terms() {
+  $alert_types = [
+    'Emergency',
+    'Fire',
+    'Flood',
+    'Medical',
+    'Lightning',
+    'Pollution',
+    'Heat wave',
+    'Traffic',
+    'Notification',
+  ];
+
+  foreach ($alert_types as $alert_type) {
+    Term::create([
+      'vid' => 'alert_type',
+      'name' => $alert_type,
+      'weight' => 100,
+    ])->save();
+  }
+}
 
 /**
  * Implements hook_install().
  */
 function tide_alert_install() {
   if (!\Drupal::service('config.installer')->isSyncing()) {
-    // Create taxonomy.
```

Comment:
What creates the vocabulary now? Does it get created when you try and add a term to it?

Reply:
@anthony-malkoun, there's a config/install/ item that creates it - https://github.com/dpc-sdp/tide_alert/blob/develop/config/install/taxonomy.vocabulary.alert_type.yml.

I'm not sure why its being manually imported in the install hook since Drupal should install it automatically. QA tested this and the vocabulary seems to be present after a clean install.